build/scripts/feature-shortcode-update/main.go

```diff
@@ -0,0 +1,158 @@
+// Copyright 2023 Google LLC All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package main implements a program that removes the contents of feature expiry version and publish version shortcodes in .md files within the site/content/en/docs directory.
+package main
+
+import (
+	"bufio"
+	"flag"
+	"log"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+func main() {
+	dirPath := "site/content/en/docs"
```

Comment:
Based on my examination of the previous releases, it appears that the shortcodes are only present in the site/content/en/docs directory. Kindly inform me if there are any other directories that require the inclusion of these shortcodes.

Reply:
Nope, that's it - you got it!